lib/workers/repository/onboarding/common.ts

```diff
@@ -0,0 +1,30 @@
+import hasha from 'hasha';
+import { configFileNames } from '../../../config/app-strings';
+import type { RenovateConfig } from '../../../config/types';
+import { logger } from '../../../logger';
+
+export const defaultConfigFile = (config: RenovateConfig): string =>
+  configFileNames.includes(config.onboardingConfigFileName!)
+    ? config.onboardingConfigFileName!
+    : configFileNames[0];
+
+export class OnboardingState {
+  private static updateRequested = false;
+
+  static get prUpdateRequested(): boolean {
+    logger.debug(
```

Comment:
i think those should be trace, as you als log something before calling the setter. so reduce the overall log messages as @rarkins already mentioned above